src/SFA.DAS.Support.Indexer.Jobs/SFA.DAS.Support.Indexer.Jobs.csproj

```diff
@@ -0,0 +1,268 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we use the new project format? These ones are ugly

Reply:
For reference https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/blob/master/src/SFA.DAS.EmployerAccounts.Jobs/SFA.DAS.EmployerAccounts.Jobs.csproj